templates/docs/examples/patterns/buttons/processing.html

```diff
@@ -4,5 +4,18 @@
 {% block standalone_css %}patterns_buttons{% endblock %}
 
 {% block content %}
-<button class="p-button--positive is-processing" disabled><i class="p-icon--spinner u-animation--spin is-light"></i></button>
+<button disabled class="p-button--positive is-processing">
+  <i class="p-icon--spinner u-animation--spin is-light"></i>
+</button>
+
+<button class="p-button--positive js-processing-button">
+    <i class="p-icon--spinner u-animation--spin is-light u-hide"></i>
```

Comment:
Accessibility wise I was going to suggest adding text/aria-label to the spinner, but if we're adding `disabled` then screen readers will ignore the whole button anyway 😅